sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/DeleteFileFn.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
```

Comment:
<!--thread_id:cc_171358974_t; commit:e3960b9fea070f4d514b71629fad156fc4f8773c; resolved:0-->
<!--section:context-quote-->
> **lgajowy** wrote:
> In such case, should `xml` module depend on `file-based-io-tests` module? My intention was to move code that is used by both modules to a place where both modules could easily access (hence the `common` module).

<!--section:body-->
I think we should move XmlIOIT to file-bsed-io-tests module as well given that all other file-based IO tests are in this module. This will allow us to consolidate all similar ITs and common classes/resources.